From foo@baz Tue Apr  9 12:12:43 2002
Date: Tue, 20 Nov 2007 13:56:21 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: driver core: clean up shutdown.c

shutdown.c had some stuff it did not need, including a duplicate extern
in the power.h file.  This cleans up all of that.


Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/power/power.h    |    7 -------
 drivers/base/power/shutdown.c |    5 -----
 2 files changed, 12 deletions(-)

--- a/drivers/base/power/power.h
+++ b/drivers/base/power/power.h
@@ -1,10 +1,3 @@
-/*
- * shutdown.c
- */
-
-extern void device_shutdown(void);
-
-
 #ifdef CONFIG_PM_SLEEP
 
 /*
--- a/drivers/base/power/shutdown.c
+++ b/drivers/base/power/shutdown.c
@@ -12,10 +12,6 @@
 #include <asm/semaphore.h>
 
 #include "../base.h"
-#include "power.h"
-
-#define to_dev(node) container_of(node, struct device, kobj.entry)
-
 
 /**
  * We handle system devices differently - we suspend and shut them
@@ -45,4 +41,3 @@ void device_shutdown(void)
 		}
 	}
 }
-
